packages/paste-website/src/pages/components/popover/index.mdx

```diff
@@ -61,8 +61,10 @@ export const pageQuery = graphql`
   <CalloutTitle>Using alpha components in your product</CalloutTitle>
   <CalloutText>
     Alpha components are ready for you to use in production. Components such as Popover are likely to not see any API
-    changes. This is only an alpha due to the lack of documentation and a matching Sketch symbol. Feel free to share
-    your experience using this component by{' '}
+    changes. This alpha component presently does not have:
+    - Complete documentation
```

Comment:
Since this is inside the Callout, we'll need to use the List component instead of markdown. It'll be something like:

```
<UnorderedList>
    <ListItem>Complete documentation</ListItem>
    <ListItem>A Sketch symbol</ListItem>
</UnorderedList>
```